# Contains data for representing the machines current position

# The current machine's true position
geometry_msgs/Vector3 machinePosition

# The current position with offsets
geometry_msgs/Vector3 workPosition
